src/url_handler.js

```diff
@@ -1,38 +1,34 @@
-import { FlashURLHandler } from './urlhandlers/flash_url_handler';
-import { NodeURLHandler } from './urlhandlers/mock_node_url_handler';
+import { flashURLHandler } from './urlhandlers/flash_url_handler';
+import { nodeURLHandler } from './urlhandlers/mock_node_url_handler';
 import { XHRURLHandler } from './urlhandlers/xhr_url_handler';
 
-export class URLHandler {
-  constructor() {
-    this.flash = new FlashURLHandler();
-    this.node = new NodeURLHandler();
-    this.xhr = new XHRURLHandler();
-  }
-
-  get(url, options, cb) {
-    // Allow skip of the options param
-    if (!cb) {
-      if (typeof options === 'function') {
-        cb = options;
-      }
-      options = {};
+function get(url, options, cb) {
+  // Allow skip of the options param
+  if (!cb) {
+    if (typeof options === 'function') {
+      cb = options;
     }
+    options = {};
+  }
 
-    if (options.urlhandler && options.urlhandler.supported()) {
-      // explicitly supply your own URLHandler object
-      return options.urlhandler.get(url, options, cb);
-    } else if (typeof window === 'undefined' || window === null) {
-      return this.node.get(url, options, cb);
-    } else if (this.xhr.supported()) {
-      return this.xhr.get(url, options, cb);
-    } else if (this.flash.supported()) {
-      return this.flash.get(url, options, cb);
-    } else {
-      return cb(
-        new Error(
-          'Current context is not supported by any of the default URLHandlers. Please provide a custom URLHandler'
-        )
-      );
-    }
+  if (options.urlhandler && options.urlhandler.supported()) {
```

Comment:
is there ever a `options.urlhandler` passed to this function? From what I saw in the code it is `fetchingOptions` only. besides in vast parser, if a custom url handler is passed, it overrides this function

Reply:
Yes probably we can remove this check